data/templates/sangoma.tmpl

```diff
@@ -64,6 +64,9 @@
         <P8684 para="DhcpVlan">{{ vlan_dhcp_enable }}</P8684>
         <!--FunctionKeys-->
         {% for idx in 1..cap_linekey_count %}
+        {% if _context["linekey_type_#{idx}"] == "queuetoggle" %}
+            {{ _context["linekey_value_#{idx}"] = queuetoggle }}
+        {% endif %}
```

Comment:
```suggestion
        {% if _context["linekey_type_#{idx}"] == "queuetoggle" %}
            {% set linekey_value = queuetoggle %}
        {% else %}
            {% set linekey_value = _context["linekey_value_#{idx}"] %}
        {% endif %}
```

Reply:
Please see the comment to the line below

Reply:
This suggestion works great too!
The code is a little bit longer and uses an additional variable always executing an assignment, also when it is useless, but the goal is equally reached!
To me is ok, you can go ahead with the modification :+1: